### DISCUSSION TITLE
S3 Storage setting page not saving

### DISCUSSION THREAD

**Discussion by zulteric** [owncast/owncast#3283] [Q&A] [answered]
Hi, I'm running a linode One Click install. ... On the S3 settings page I'm trying to enter my bucket settings but the Save button is grayed out and will not let me save anything I enter. I've tried rebooting the whole instance but that did not help.

I'm unsure how to go about this another way - maybe I can manually enter the information in a config file?

**Comment by gabek** [comment]
The only thing I can think of is that a required value isn't present. Are they all filled in? Can you show an example of what your form looks like?

**Comment by zulteric** [comment]
![Screenshot(1)](https://github.com/owncast/owncast/assets/9518351/e78b1fa4-f0fa-42a4-b118-5377acacc01e)

**Comment by zulteric** [comment, marked as answer]
Didn't put in https:// thank you!
